.github/workflows/mdx-lint.yml

```diff
@@ -2,7 +2,7 @@ name: MDX Lint
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, aj/md-lint ]
```

Comment:
Added to test it live.

Reply:
Interesting. I thought that CI would be triggered anyway because of 

```
pull_request:
    branches: [ master ]
```

Reply:
There is no `master` branch in this repository - this (and below) should be `main`

Reply:
Good point! I thought we still used `master` for historical reasons. Then we probably want something like:

```yml
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```